From 5de1d54924cd0dae3f13f2c8ee61526a97de0999 Mon Sep 17 00:00:00 2001
From: Dave Airlie <airlied@redhat.com>
Date: Fri, 12 Sep 2014 01:56:55 -0400
Subject: [drm] mst: fix build with debugfs off

Message-id: <1410487019-24661-18-git-send-email-airlied@redhat.com>
Patchwork-id: 93925
O-Subject: [RHEL7.1 BZ1140440 PATCH 17/21] drm/mst: fix build with debugfs off.
Bugzilla: 1140440
RH-Acked-by: Jerome Glisse <jglisse@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Rob Clark <rclark@redhat.com>

bugzilla: 1140440
upstream: 75bc08ab276f3e6f436847328631a07f30f973c4

seq_file.h was being pulled in via drm_mm.h in that case.

Reported-by: Matthew Thode (on irc)
Signed-off-by: Dave Airlie <airlied@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/gpu/drm/drm_dp_mst_topology.c b/drivers/gpu/drm/drm_dp_mst_topology.c
index df89128..97e3922 100644
--- a/drivers/gpu/drm/drm_dp_mst_topology.c
+++ b/drivers/gpu/drm/drm_dp_mst_topology.c
@@ -25,6 +25,7 @@
 #include <linux/init.h>
 #include <linux/errno.h>
 #include <linux/sched.h>
+#include <linux/seq_file.h>
 #include <linux/i2c.h>
 #include <drm/drm_dp_mst_helper.h>
 #include <drm/drmP.h>
-- 
1.7.1